src/lib/dvf/token/getTokenInfoByTokenId.js

```diff
@@ -0,0 +1,23 @@
+const DVFError = require('../DVFError')
+
+module.exports = (dvf, starkTokenId) => {
+  const {tokenRegistry} = dvf.config
+
+  if (!tokenRegistry) {
+    throw new DVFError('NO_TOKEN_REGISTRY')
+  }
+  let tokenInfo
+  Object.keys(tokenRegistry)
+    .forEach((token) => {
```

Comment:
find would be more appropriate https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find